taxi/urls.py

```diff
@@ -13,7 +13,10 @@
     ManufacturerDeleteView,
     DriverCreateView,
     DriverDeleteView,
-    DriverLicenseUpdateView, CarListView, CarDetailView, CarDriversUpdateView,
+    DriverLicenseUpdateView,
+    CarListView,
```

Comment:
Keep at least 1st letter alphabetical order in imports, please

Reply:
Fixed that in new commit in 4 files